packages/apps/todomvc/src/components/Main.tsx

```diff
@@ -145,14 +119,14 @@ export const Main = () => {
             Share
           </button>
           <ul className='todo-list'>
-            {shownTodos.map((todo) => (
+            {todos.map((todo) => (
               <TodoItem
-                key={todo.id}
+                key={todo._id}
                 todo={todo}
                 onToggle={() => handleToggle(todo)}
                 onDestroy={() => handleDestroy(todo)}
-                onEdit={() => setEditing(todo.id)}
-                editing={editing === todo.id}
+                onEdit={() => setEditing(todo._id)}
```

Comment:
use `todo[id]` instead

we probably can hide underscored properties from typescript